man/en/sos-report.1

```diff
@@ -313,7 +313,7 @@ a timeout for a specific plugin, use the 'timeout' plugin option available to
 all plugins - e.g. '-k logs.timeout=600'.
 
 The plugin-specific timeout option will override this option. For example, using
-\'--plugin-timeout=60 -k logs.timeout=600\' will set a timeout of 600 seconds for
+'--plugin-timeout=60 -k logs.timeout=600' will set a timeout of 600 seconds for
```

Comment:
This came up with the following issue, and hence the change

```
I: sosreport: acute-accent-in-manual-page [usr/share/man/man1/sos-
report.1.gz:313]
N: 
N:   This manual page uses the \' groff sequence. Usually, the intent is to
N:   generate an apostrophe, but that sequence actually renders as an acute
N:   accent.
N:   
N:   For an apostrophe or a single closing quote, use plain '. For single
N:   opening quote, i.e. a straight downward line ' like the one used in
shell
N:   commands, use '\(aq'.
N:   
N:   In case this tag was emitted for the second half of a '\\' sequence,
this
N:   is indeed no acute accent, but still wrong: A literal backslash should
be
N:   written \e in the groff format, i.e. a '\\' sequence needs to be changed
N:   to '\e' which also won't trigger this tag.
N: 
N:   Please refer to Bug#554897, Bug#507673, and Bug#966803 for details.
N: 
N:   Visibility: info
N:   Show-Always: no
N:   Check: documentation/manual
N:   Renamed from: acute-accent-in-manpage
N: 
N:
I: sosreport: acute-accent-in-manual-page
[usr/share/man/man1/sosreport.1.gz:313]
```

Below is what was seen in the man page

```
using ´--plugin-timeout=60 -k logs.timeout=600´ will
```